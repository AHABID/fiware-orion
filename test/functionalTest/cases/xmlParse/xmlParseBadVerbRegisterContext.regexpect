HTTP/1.1 405 Method Not Allowed
Content-Length: 0
Allow: POST
Date: REGEX(.*)

++++++++++++++++++++
