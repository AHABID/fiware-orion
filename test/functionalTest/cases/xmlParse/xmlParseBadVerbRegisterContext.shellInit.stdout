{ "dropped" : "testharness", "ok" : 1 }
T@17:45:27  paConfigActions[727]: setting trace levels to '0-255'
T@17:45:27  paConfigActions[728]: Leaving function paConfigActions
E@17:45:27  exitFunc[1009]: unlink(/tmp/orion_9999.pid): No such file or directory
T@17:45:27  semInit[61]: Initialized 'req' semaphore
T@17:45:27  semInit[62]: Initialized 'mongo' semaphore
T@17:45:27  mongoSemTake[90]: mongoConnect taking the 'mongo' semaphore for 'connecting to mongo'
T@17:45:27  mongoSemTake[92]: mongoConnect has the 'mongo' semaphore
T@17:45:27  mongoConnect[152]: mongo version server: 2.4.9 (mayor: 2, minor: 4, extra: 9)
T@17:45:27  mongoSemGive[118]: mongoConnect gives the 'mongo' semaphore for 'connecting to mongo'
F@17:45:27  ***[1063]: Connected to mongo at localhost:testharness
T@17:45:27  ensureLocationIndex[381]: ensuring 2dsphere index on location.coords (tenant )
T@17:45:27  recoverOntimeIntervalThreads[398]: query() in 'testharness.csubs' collection: '{ conditions.type: "ONTIMEINTERVAL" }'
T@17:45:27  mongoSemTake[90]: recoverOntimeIntervalThreads taking the 'mongo' semaphore for 'query in SubscribeContextCollection'
T@17:45:27  mongoSemTake[92]: recoverOntimeIntervalThreads has the 'mongo' semaphore
T@17:45:27  mongoSemGive[118]: recoverOntimeIntervalThreads gives the 'mongo' semaphore for 'query in SubscribeContextCollection'
V@17:45:27  restStart[569]: Starting HTTP daemon on IPv4 0.0.0.0 port 9999
V@17:45:27  restStart[611]: Starting HTTP daemon on IPv6 :: port 9999
T@17:45:28  connectionTreat[438]: notifyFormat: ''
T@17:45:28  httpHeaderGet[137]: HTTP Header:   User-Agent: curl/7.26.0
T@17:45:28  httpHeaderGet[137]: HTTP Header:   Host: localhost:9999
T@17:45:28  httpHeaderGet[137]: HTTP Header:   Content-Type: application/xml
T@17:45:28  httpHeaderGet[137]: HTTP Header:   Accept: application/xml
T@17:45:28  httpHeaderGet[137]: HTTP Header:   Content-Length: 1155
T@17:45:28  connectionTreat[442]: HTTP tenant: ''
T@17:45:28  connectionTreat[470]: 'default' value for notifyFormat (ciP->outFormat == 1)): 'XML'
T@17:45:28  restService[197]: Treating service * /ngsi10/subscribeContext
T@17:45:28  restService[237]: tenant: ''
W@17:45:28  badVerbPostOnly[48]: bad verb for url '/ngsi10/subscribeContext', method 'BAD_VERB'
T@17:45:28  restReply[65]: Response 1: responding with 0 bytes, Status Code 405
T@17:45:28  restReply[66]: Response payload: ''
T@17:45:28  ~RegisterContextResponse[56]: destroyed
F@17:45:28  ***[981]: In sigHandler - caught signal 15
X@17:45:28  sigHandler[987]: Received signal 15
